api/package.json

```diff
@@ -35,6 +35,7 @@
         "pino": "^8.11.0",
         "pino-http": "^8.3.3",
         "pino-pretty": "^10.0.0",
+        "powerhouse-switchboard-plugin-demo": "file:../module-example",
```

Comment:
Can you please make it consistent: either call both things `plugin-demo` or `module-example`. I suggest to use `module-example` 🙂